js/NextPeriodLineItemHandler.js

```diff
@@ -222,7 +222,7 @@ function createLineItem(params) {
         null,
         'success'
       );
-
+      createActivity('Update Payment Plan Next Period', 'Update Payment Plan Next Period');
```

Comment:
I see the second parameter for the createActivity method is typeID. Are you sure 'Pdate Payment Plan Next Period' can be used as activity type ID? Shouldn't it be 'update_payment_plan_next_period'? Same for all other calls to that method.